src/components/views/messages/MFileReplyBody.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020-2021 Tulir Asokan <tulir@maunium.net>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from "react";
+
+import MFileBody from "./MFileBody";
+import { _t } from "../../../languageHandler";
+import AccessibleButton from "../elements/AccessibleButton";
+import { presentableTextForFile } from "../../../utils/FileUtils";
+
+export default class MFileReplyBody extends MFileBody {
```

Comment:
it's also not clear what this component's purpose is, and it appears to duplicate the code

Reply:
as i didn't find a way to make the MFileBody component aware that it is in a reply, I use this component to override the onPlaceholderClick function and prevent download on click
i will try another way adding a property "isInReply" in IBodyProps to remove this duplication